Assets/Scripts/Drone/ScannedThingsLog/ShowNewlyScannedThing.cs

```diff
@@ -8,6 +8,8 @@
 /// </summary>
 public class ShowNewlyScannedThing : MonoBehaviour
 {
+    public static ShowNewlyScannedThing instance;
```

Comment:
This should be 
public static ShowNewlyScannedThing Instance {get; private set;}